box2d_lib.pri

```diff
@@ -1,7 +1,7 @@
 INCLUDEPATH += $$PWD
 
-contains(DEFINES, BOX2D_SYSTEM) {
-
+contains(CONFIG, BOX2D_SYSTEM) {
+    message(Building with system Box2d)
```

Comment:
I guess we have to remove this line. It's so annoying. :smile: If you run your project in Qt Creator each time you compile it the editor switches to `Messages` tab. I think this function useful for warnings not for regular messages.